src/avram/migrator/runner.cr

```diff
@@ -127,13 +132,17 @@ class Avram::Migrator::Runner
     SQL
   end
 
-  def self.run(command : String, output : IO = STDOUT)
+  def self.run(command : String, output : IO = STDOUT, input : Process::Stdio = Process::Redirect::Close)
+    program, *args = command.split(' ')
```

Comment:
I don't think you'd ever have spaces in any of these, but just in case, I'm going to move these over to passing in an array of args directly.